docs/getting_started/resources/talks.md

```diff
@@ -1,6 +1,6 @@
 # Talks and slides
 
-You can find curated list of talks and slides:
+BIDS-related talks and presentations are ongoing. Two great resources for curated lists of talks and slides are:
```

Comment:
```suggestion
BIDS-related talks and presentations are curated and updated on an ongoing basis in 2 key locations and the community is invited to consult and re-use them here:
```

Reply:
adding the missing angle that we would really like people to use these semi-official decks